### PR TITLE
drivers: Fix dw_mmc

### PR DESCRIPTION
**Pull request by anton-bondarev**

### CODE REVIEW

File: src/drivers/mmc/host/dw_mmc/dw_mmc_socfpga.c [hunk 1]
```diff
@@ -1090,11 +1093,17 @@ static int dw_mci_init_slot(struct dw_mci *host, unsigned int id) {
 	uint32_t freq[2];
 #endif
 
-	mmc = mmc_alloc_host(sizeof(struct dw_mci_slot), NULL);
+	//mmc = mmc_alloc_host(sizeof(struct dw_mci_slot), NULL);
```
Comment (Deryugin) [inline src/drivers/mmc/host/dw_mmc/dw_mmc_socfpga.c]: Delete this line, please

File: src/drivers/mmc/host/dw_mmc/dw_mmc_socfpga.c [hunk 2]
```diff
@@ -40,6 +40,8 @@ static inline void udelay(int i) {
 
 #define BASE_ADDR OPTION_GET(NUMBER, base_addr)
 #define IRQ_NUM   OPTION_GET(NUMBER, irq_num)
+#define SLOTS_QUANTITY OPTION_GET(NUMBER, slots_quantity)
+
```
Comment (Deryugin) [inline src/drivers/mmc/host/dw_mmc/dw_mmc_socfpga.c]: Double blank line

Comment (anton-bondarev) [reply src/drivers/mmc/host/dw_mmc/dw_mmc_socfpga.c]: done

File: src/drivers/mmc/host/dw_mmc/dw_mmc_socfpga.c (hunk 1 above)
Comment (anton-bondarev) [reply src/drivers/mmc/host/dw_mmc/dw_mmc_socfpga.c]: done

**Pull request merged**